File: src/B_2845/B_2845.java
package B_2845;

import java.io.*;
import java.util.*;

public class B_2845 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        StringBuilder sb = new StringBuilder();
        int L = Integer.parseInt(st.nextToken())*Integer.parseInt(st.nextToken());
        st = new StringTokenizer(br.readLine());
        for(int i=0; i<5; ++i){
            sb.append(Integer.parseInt(st.nextToken()) - L).append(" ");
        }
        System.out.println(sb);
    }
}

File: src/B_2845/B_2845.kt
package B_2845

import java.io.*
import java.util.*

fun main()=with(BufferedReader(InputStreamReader(System.`in`))){
    var st = StringTokenizer(readLine())
    val sb = StringBuilder()
    val L = st.nextToken().toInt() * st.nextToken().toInt()
    st = StringTokenizer(readLine())
    for (i in 0..4) {
        sb.append(st.nextToken().toInt() - L).append(" ")
    }
    println(sb)
}